backend/plugin/db/dm/dm.go

```diff
@@ -59,6 +60,7 @@ func (driver *Driver) Open(_ context.Context, _ storepb.Engine, config db.Connec
 	}
 	driver.db = db
 	driver.databaseName = config.Database
+	driver.maximumSQLResultSize = config.MaximumSQLResultSize
```

Comment:
We wanna use the default the value?

Reply:
We will insert the default value into the setting, also fallback to the default value if cannot get the value from the store